hwy/ops/rvv-inl.h

```diff
@@ -1816,9 +1876,9 @@ HWY_API void StoreN(size_t count, VFromD<D> v, D d, TFromD<D>* HWY_RESTRICT p) {
 template <class D>
 HWY_API void StoreN(VFromD<D> v, D d, TFromD<D>* HWY_RESTRICT p,
```

Comment:
Do we also want to apply capping to LoadN?

Reply:
The capping is already applied to the LoadN operation through `CappedLanes(d, num_lanes)` in the LoadN implementation.